frame/contracts/src/benchmarking.rs

```diff
@@ -0,0 +1,273 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Benchmarks for the contracts pallet
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::*;
+use crate::Module as Contracts;
+
+use frame_benchmarking::{benchmarks, account};
+use frame_system::{Module as System, RawOrigin};
+use parity_wasm::elements::FuncBody;
+use sp_runtime::traits::Hash;
+
+const SEED: u32 = 0;
```

Comment:
Why do we need this one?

Reply:
Strictly speaking we do not need it. It is a pattern I copied from the benchmarks which also use a `SEED` constant that is then passed to `pallet_benchmarking::account` which uses it to generate a `AccoundId`. It basically does not matter which seed to use because the storage is deleted anyways before each benchmark.

I considered it a best practice for benchmarking. Maybe @shawntabrizi can elaborate.

Reply:
that's fine, although a dangling `SEED` asking for question. I wonder if we could move it where it actually is used and/or leave a comment for posterity.

Reply:
It is used in the `create_funded_user` function.

Reply:
I just removed it and used `0` there.